CMSgov/design-system
# [WNMGDS-44] Disable firefox styles for :invalid pseudo-class

## Summary
To stay consistent across browsers of not applying a style for :invalid pseudo-class, this PR disables the red "glow" of [Gecko default](https://developer.mozilla.org/en-US/docs/Web/CSS/:invalid#gecko_defaults)

## How to test
- Add this code snippet on one of examples on doc site

```
<form>
<input class="ds-c-field ds-c-field--medium" id="input-medium" type="email" required="">
<input type="submit" value="Submit">
</form>
```

- Go to doc site using firefox, you should not be able to see the red 'glow' on error
